database/seeds/UserSeeder.php

```diff
@@ -17,5 +18,14 @@ public function run()
         $user->email = 'ha@gmail.com';
         $user->password = bcrypt('123456');
         $user->save();
+                
+        for ($i=0; $i < 3; $i++) { 
```

Comment:
```suggestion
        for ($i = 0; $i < config('seeder.user_limit'); $i++) { 
```
Mấy số như vậy bỏ vào file config nhé